NMDSdevopsServiceAdm/DataEngineering
# Purge outdated workplace data

# Description
Purge outdated ascwds workplace data

# How to test
Tested in branch and checked the processed data - new column was added and after checking a few of the rows with "purge_data=true" it seems to be doing what it should be 

Feel free to double check here: https://eu-west-2.console.aws.amazon.com/gluestudio/home?region=eu-west-2#/editor/job/purge-outdated-workplace-data-clean_ascwds_workplace_data_job/runs

[Bucket with cleaned data](https://s3.console.aws.amazon.com/s3/buckets/sfc-purge-outdated-workplace-data-datasets?region=eu-west-2&bucketType=general&prefix=domain%3DASCWDS/dataset%3Dworkplace_cleaned/year%3D2024/month%3D02/day%3D15/import_date%3D20240215/&showversions=false)

[Ticket](https://trello.com/c/MyES0K1r/45-epic-5-purge-and-tidy-up-file-for-pipeline-using-update-date-must)
# Developer checklist
